Cargo.toml

```diff
@@ -13,3 +13,4 @@ exclude = [ "/examples" ]
 tauri = "1"
 serde_json = "1.0"
 tiny_http = "0.11"
+http = "0.2"
```

Comment:
I feel like we should use the [url crate](https://crates.io/crates/url) instead. What do you think? (Not sure about the differences in this case but url is a smaller crate 🤷)

Reply:
1) url crate cannot parse relative url, will return RelativeUrlWithoutBase error. 2) tauri self already uses http crate.